R/fsi_functions.R

```diff
@@ -62,6 +89,45 @@ fsi_add_fsa <- function(fsi, lvar, tbl) {
   fsi
 }
 
+
+
+#' @title fsi_add_cs
+#' 
+#' @description fsi_add_cs adds the consequent to the FSI model
+#' 
+#' @usage 
+#' 
+#' fsi_add_cs(fsi, lvar, lvals, mfs, bounds)
+#' 
+#' @param fsi The FSI model instantiated with the `fsi_create` function.
+#' @param lvar A character value that represents a linguistic variable of the consequent.
+#' @param lvals A character vector that represents linguistic values of the linguistic variable of the consequent.
+#' @param mfs A character vector of corresponding membership functions.
+#' @param bounds A numeric vector that represents the lower and upper bounds of the consequent domain. 
+#' 
+#' @details 
+#' 
```

Comment:
Please, provide details about the format of the parameters `lvals`, `mfs` and `bounds`. For instance, there is a correspondence between the character vector `lvals` and the function vector `mfs`. This means that their length must be equal.